app/src/main/java/net/skyscanner/backpack/demo/data/ComponentRegistry.kt

```diff
@@ -361,6 +362,7 @@ object ComponentRegistry {
         TAB_TITLE_COMPOSE composeStory { FlareStory() },
       ),
     ),
+    "FlightLeg" composeStory { FlightLegStory() },
```

Comment:
massive nitpick: can you add a space here? (also need to update the `DocsRegistry` then)